content/board_members_dwh.yml

```
@@ -1,19 +1,31 @@
 board:
 
-  - name: Rory Baart
+  - name: Dries Stuij
     description:
       nl: Voorzitter
       en: President
     email: voorzitter@dwhdelft.nl
     
-  - name: Casper Boone
+  - name: Vanadie
     description:
-      nl: Interim Secretaris
-      en: Interim Secretary
+      nl: Secretaris
+      en: Secretary
     email: secretaris@dwhdelft.nl
 
-  - name: Dries Stuij
+  - name: Suzanne Kleijweg
     description:
       nl: Penningmeester
       en: Treasurer
     email: penningmeester@dwhdelft.nl
+    
+  - name: Casper Boone
+    description:
+      nl: Algemeen bestuurslid
+      en: General board member
```

```suggestion
      en: Board member
```
Misschien gewoon "board member"?